public/app/plugins/datasource/loki/modifyQuery.ts

```diff
@@ -574,9 +588,19 @@ function getAllPositionsInNodeByType(query: string, node: SyntaxNode, type: numb
   let pos = 0;
   let child = node.childAfter(pos);
   while (child) {
-    positions.push(...getAllPositionsInNodeByType(query, child, type));
+    positions.push(...getAllPositionsInNodeByType(child, type));
     pos = child.to;
     child = node.childAfter(pos);
   }
   return positions;
 }
```

Comment:
All those changes above are basically just chores. Let me know if I should remove them.